continuous_eval/metrics/code/sql/sql_deterministic_metrics.py

```diff
@@ -0,0 +1,77 @@
+from typing import List, Union
+
+import sqlparse
```

Comment:
Can avoid sqlparse to format the query? Can we use sqlglot only. It seems sqlglot can format but also optimize in a common format https://github.com/tobymao/sqlglot?tab=readme-ov-file#sql-optimizer